query_V1

```diff
@@ -11,43 +26,154 @@
+				transition: 0.15s ease-in-out;
+				cursor: pointer;
+			}
+			
+			.index:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
+			.index {transition: 0.6s;}
+			
+			.settings:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
+			.settings {transition: 0.6s;}
+			
+			.sign-out:hover {transform: scale(1.1, 1.1); transition: 0.6s;}
+			.sign-out {transition: 0.6s;}
+			
 		</style>
 		
 	</head>
 	<body>
 		
 		<div class="grid-container">
+			<?php
+				//Pulls the links from the nav.php page and places them in the navigation div
+				require 'nav.php'; //'require' is 100% needed for the site to run 
+			?>
 			
-			<div class="sidenav">
-				
-				<p style="color: white; position: fixed; top: 0vw;">Test</p>
-				
-				<p style="color: white; position: fixed; top: 2vw;">Test</p>
-				
-				<p style="color: white; position: fixed; top: 4vw;">Test</p>
-				
-				<p style="color: white; position: fixed; top: 6vw;">Test</p>
-			
-			</div>
+			<?php
+				require 'loading.php';
+			?>
 			
-			<div class="banner"></div>
+			<?php
+							require "Database_mysqli.php";
+
+							//Creates a varible to store the sql query
+							$query_2 = ("SELECT SEC_TO_TIME(SUM(s.Duration)) AS 'Total'
+							FROM Songs AS s 
+
+							INNER JOIN Album_PK a ON s.Album_FK = a.Album_PK
+
+							JOIN ID_To_Genre j1 ON s.Song_ID = j1.Song_ID
+							JOIN Genre_PK g ON g.Genre_PK = j1.Genre_FK
+
+							JOIN ID_To_Artist j2 ON s.Song_ID = j2.Song_ID
+							JOIN Artist_PK ar ON ar.Artist_PK = j2.Artist_FK
+
+							WHERE s.title LIKE '%$searched%' or g.Genre LIKE '%$searched%' or ar.Artist LIKE '%$searched%' or a.Album LIKE '%$searched%' or s.Duration LIKE '%$searched%'");
+
+							//Runs and stores the query using the varibles $con (see nav.php) and $query
+							$total = mysqli_query($conn,$query_2);
+							//runs in a 'while' loop
+							while($output_2=mysqli_fetch_array($total))
+							{
+							
+						?>
 			
-			<div class="strip" style="position: sticky; top: 0vw;">
+			<div id="top" style="grid-row: 1; grid-column: 1;"></div>
 			
-				<p style="color: white;">Test</p>
+			<div class="banner" style="background-image: url('images/banner_search.jpg'); background-size: 80vw 18vw; z-index: 100;"></div>
 			
+			<div class="strip" style="position: sticky; top: 0vw; background-image: url('images/banner_search.jpg'); background-size: 80vw 18vw; background-position: 0vw 8vw; z-index: 100;">
+
+				<div style="margin: 2vw 3vw 0vw 5vw; height: 2vw; width: 50vw; background-color: transparent;">
+
+					<h3 style="margin: -0.1vw 0vw; float: left; color: white; font-size: 1.9vw; font-family: 'Noto Sans JP', sans-serif;">Dynamic Query</h3>
+
+				</div>
+				<div style="margin: 0.5vw 5vw; height: 1vw; width: 12vw; background-color: transparent;">
+
+					<h5 style="font-family: 'Noto Sans JP', sans-serif; margin: -0.15vw 0vw; width: 40vw; float: left; color: white; font-size: 1vw;">Search Results For "<?php echo $searched ?>"</h5>
+
```

Comment:
Here I have added the divs that will be used as the header for the query page. It is split into two to allow for a sticky bar.